### PR TITLE
Added Crowd Authentication plugin references

### PR DESCRIPTION
**Pull request by mgambini**
Hi,
has anyone checked the plugin?

### CODE REVIEW

**Comment by varshavaradarajan** [comment]
Hi, 
will pick it up today.

File: data/authorization_plugins.yml [hunk 1]
```diff
@@ -21,4 +21,13 @@
...
   contact_url: mailto:support@thoughtworks.com?subject=Info about LDAP/AD Authentication Plugin
   paid: true
+  
+- name: Atlassian Crowd Authentication Plugin
+  description: Delegates authentication services to Atlasian Crowd
+  readmore_url: https://gitlab.com/mgambini1975/go-crowd-plugin.git
+  author_url: http://www.ditechonline.it/
+  author_name: Di.Tech S.p.A.
+  contact_url: mailto:marco.gambini@ditechspa.it?subject=Info about Crowd Authentication Plugin
+  paid: false
```
Comment (varshavaradarajan) [inline data/authorization_plugins.yml]: There should be a releases_url. The idea is that users can just download the jar in server plugins external. Check the other yaml files. The reason the authorization plugins don't have the releases url is that its already bundled with go and users don't have to download it.

**Comment by varshavaradarajan** [comment]
@mgambini , checking the README, it feels like one has to edit the crowd.properties and then build a jar. An easier experience for the user would be to download the jar straightaway and edit this file. Please look at  https://github.com/ashwanthkumar/gocd-slack-build-notifier as to how you can source the conf file. Can you please elaborate on the properties that is intended to be configured? Its optional. But might be nice to have. Your call.

File: data/authorization_plugins.yml (hunk 1 above)
Comment (mgambini) [inline data/authorization_plugins.yml]: Do you mean I should commit the plugin on github.com (now source code is hosted on gitlab.com) and use github continuous integration?

Comment (varshavaradarajan) [inline data/authorization_plugins.yml]: @mgambini , you can use any CI tool and any SCM platform as long you you can provide a jar that can be downloadable by users.

**Comment by mgambini** [comment]
@varshavaradarajan, I moved source code to GitHub, activated CI and published a first release: https://github.com/mgambini/crowd-integration-plugin/releases

Also I updated the data/authorization_plugins.yml file adding releases_url info.
Sincerely,
Marco

**Comment by mgambini** [comment]
@varshavaradarajan, have you picked up my changes?
Cheers,
Marco

**Pull request merged**